pkg/connector/chatinfo.go

```diff
@@ -493,12 +502,16 @@ func (t *TelegramClient) getPowerLevelOverridesFromBannedRights(entity tg.ChatCl
 		event.StateBeeperDisappearingTimer: 85,
 	}
 
-	if dbr.ChangeInfo {
+	// If a flag such as ChangeInfo is _true_, then that means the user is banned from changing that set of fields
+	if !dbr.ChangeInfo {
+		// Otherwise, set the more permissive power levels
 		plo.Events[event.StateRoomName] = *changeInfoPowerLevel
 		plo.Events[event.StateRoomAvatar] = *changeInfoPowerLevel
 		plo.Events[event.StateTopic] = *changeInfoPowerLevel
+
 		// TODO is this the correct level?
-		plo.Events[event.StateBeeperDisappearingTimer] = *changeInfoPowerLevel
+		// any user in a group chat / supergroup appears to be able to change the disappearing timer, not just admins
```

Comment:
Any user can change it even when info changes are restricted? 🤔

Reply:
the boolean is the opposite.

https://core.telegram.org/constructor/chatBannedRights

>  change_info | flags.10?true | If set, does not allow any user to change the description of a supergroup/chat

If ChatInfo is true, then the user is "banned" or "not allowed" to edit, so we leave the power levels out and just default to the superadminPowerLevel of StateDefault above.

If it's false then we set more reasonable power levels.

At least this is what i saw in my local bridge I think